ChessBoardApp/ChessBoard/InputValidation.cs

```diff
@@ -0,0 +1,42 @@
+using Serilog;
+
+namespace ChessBoardApp.ChessBoard
+{
+
+
+    public class InputValidation
+    {
+        private readonly ILogger _logger;
+
+        public InputValidation(ILogger logger)
+        {
+            _logger = logger;
+        }
+        public class ValidationResult
+        {
+            public bool IsValid { get; set; }
+            public string ErrorMessage { get; set; }
```

Comment:
We could use nullable string here to avoid warning messages

Reply:
adjusted